__tests__/app.test.js

```diff
@@ -116,3 +116,161 @@ describe('get articles',()=>{
   });
 
 })
+
+describe('POST /api/articles/:article_id/comments',()=>{
+  test("return 201 and newly posted comment", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: 'this is my sensible and nice comment'
+    }
+    return request(app)
+      .post("/api/articles/6/comments")
```

Comment:
great testing, one more for post could be a 201 that ignores unnecessary properties